Resources/Prototypes/Maps/game.yml

```diff
@@ -86,3 +90,49 @@
     MedicalDoctor: [ 1, 1 ]
     ResearchDirector: [ 1, 1 ]
     Botanist: [ 1, 1 ]
+
+- type: gameMap
+  id: sunrise
+  mapName: 'Sunrise'
+  mapNameTemplate: '{0} Sunrise {1}'
+  nameGenerator:
+    !type:NanotrasenNameGenerator
+    prefixCreator: 'SS'
+  mapPath: Maps/sunrise.yml
+  minPlayers: 0
+  maxPlayers: 8
+  overflowJobs: []
+  availableJobs:
+    Bartender: [ 1, 1 ]
+    Captain: [ 1, 1 ]
+    ChiefEngineer: [ 1, 1 ]
+    StationEngineer: [ 1, 1 ]
+    ChiefMedicalOfficer: [ 1, 1 ]
+    MedicalDoctor: [ 1, 1 ]
+    ResearchDirector: [ 1, 1 ]
+    Botanist: [ 1, 1 ]
+
```

Comment:
Not a map

Reply:
That's from another branch of mine, dunno how it got there.
Removed.